Dockerfile

```diff
@@ -0,0 +1,30 @@
+# adapted from https://github.com/opsani/servo-ec2asg-newrelic/blob/master/Dockerfile
+FROM python:3.6-slim
+
+WORKDIR /servo
+
+# Install dependencies
+RUN pip3 install requests PyYAML python-dateutil boto3
+
+# Change /usr/bin/python3 => /usr/local/bin/python3, since the first is older,
```

Comment:
Is boto3 needed? This /usr/bin/python3 => /usr/local/bin/python3 should not be needed either.